src/main/java/org/apache/maven/plugins/dependency/filters/AbstractDependencyFilter.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.maven.plugins.dependency.filters;
```

Comment:
Is this introducing a new package? Probably not needed.

Reply:
Yup, because code is similar to artifact-filter's ArtifactFilter which is also in its own package. Also, more filters could be introduced later. Is there a downside of creating a package I do not know?

Reply:
It tends to force methods to be public for use by other subpackages that otherwise wouldn't have to be public.

Reply:
@elharo is this change a requirement for this PR to be accepted? I agree with @bmarwell approach (to be future proof), but if it is mandatory, I think it is possible to change packages.

Reply:
Yes, packaghes are overused with bad consequences